src/util/translation.h

```diff
@@ -16,6 +16,11 @@
 struct bilingual_str {
     std::string original;
     std::string translated;
+
+    bool empty() const
+    {
+        return original.empty() && translated.empty();
```

Comment:
```suggestion
        return original.empty();
```

I think translators should not be able to influence executions paths by blanking messages

Reply:
> I think translators should not be able to influence executions paths by blanking messages

Sure!

Reply:
[Update](https://github.com/bitcoin/bitcoin/pull/18927#issuecomment-626321249).